src/core/solver-base.cpp

```diff
@@ -100,6 +102,33 @@ double SolverAbstract::computeDynamicFeasibility() {
   return tmp_feas_;
 }
 
+double SolverAbstract::computeEqualityFeasibility() {
```

Comment:
Perhaps we should generalise this to `computeConstraintFeasibility` to do both equality and inequality in the same go?

Reply:
No, we cannot. Typically solvers need to differentia both. I will push a function for inequality constraints.